llm/main.py

```diff
@@ -34,6 +34,10 @@ def build_callback(name, kwargs):
         return SpeedMonitorMFU(
             window_size=kwargs.get('window_size', 1),
             gpu_flops_available=kwargs.get('gpu_flops_available', None))
+    elif name == 'optimizer_monitor':
+        return OptimizerMonitor(
+            log_optimizer_metrics=kwargs.get('log_optimizer_metrics'),
```

Comment:
Do we want to set a default ie `log_optimizer_metrics=kwargs.get('log_optimizer_metrics', DEFAULT)`?
(The [default in the callback definition](https://github.com/mosaicml/composer/blob/dev/composer/callbacks/optimizer_monitor.py#L85) is `True`)